src/languages/en.ts

```diff
@@ -2035,38 +2049,49 @@ export default {
+        billableExpense: 'Billable no longer valid',
+        cashExpenseWithNoReceipt: ({amount}: ViolationsCashExpenseWithNoReceiptParams) => `Receipt required over ${amount}`,
+        categoryOutOfPolicy: 'Category no longer valid',
+        conversionSurcharge: ({surcharge}: ViolationsConversionSurchargeParams) => `Applied ${surcharge}% conversion surcharge`,
+        customUnitOutOfPolicy: 'Unit no longer valid',
+        duplicatedTransaction: 'Potential duplicate',
+        fieldRequired: 'Report fields are required',
+        futureDate: 'Future date not allowed',
+        invoiceMarkup: ({invoiceMarkup}: ViolationsInvoiceMarkupParams) => `Marked up by ${invoiceMarkup}%`,
+        maxAge: ({maxAge}: ViolationsMaxAgeParams) => `Date older than ${maxAge} days`,
+        missingCategory: 'Missing category',
+        missingComment: 'Description required for selected category',
+        missingTag: ({tagName}: ViolationsMissingTagParams) => `Missing ${tagName ?? 'tag'}`,
+        modifiedAmount: 'Amount greater than scanned receipt',
+        modifiedDate: 'Date differs from scanned receipt',
+        nonExpensiworksExpense: 'Non-Expensiworks expense',
+        overAutoApprovalLimit: ({formattedLimitAmount}: ViolationsOverAutoApprovalLimitParams) => `Expense exceeds auto approval limit of ${formattedLimitAmount}`,
+        overCategoryLimit: ({categoryLimit}: ViolationsOverCategoryLimitParams) => `Amount over ${categoryLimit}/person category limit`,
+        overLimit: ({amount}: ViolationsOverLimitParams) => `Amount over ${amount}/person limit`,
+        overLimitAttendee: ({amount}: ViolationsOverLimitParams) => `Amount over ${amount}/person limit`,
+        perDayLimit: ({limit}: ViolationsPerDayLimitParams) => `Amount over daily ${limit}/person category limit`,
+        receiptNotSmartScanned: 'Receipt not verified. Please confirm accuracy.',
+        receiptRequired: ({amount, category}: ViolationsReceiptRequiredParams) => `Receipt required over ${amount} ${category ? ' category limit' : ''}`,
+        rter: ({brokenBankConnection, email, isAdmin, isTransactionOlderThan7Days, member}: ViolationsRterParams) => {
+            if (brokenBankConnection) {
+                return isAdmin
+                    ? `Can't auto-match receipt due to broken bank connection which ${email} needs to fix`
+                    : "Can't auto-match receipt due to broken bank connection which you need to fix";
+            }
+            if (!isTransactionOlderThan7Days) {
+                return isAdmin ? `Ask ${member} to mark as a cash or wait 7 days and try again` : 'Awaiting merge with card transaction.';
+            }
+
+            return '';
+        },
+        smartscanFailed: 'Receipt scanning failed. Enter details manually.',
+        someTagLevelsRequired: 'Missing tag',
+        tagOutOfPolicy: ({tagName}: ViolationsTagOutOfPolicyParams) => `${tagName ?? ''} no longer valid`,
```

Comment:
If `tagName` is not available we should have fallen back to `Tag` instead of an empty string for consistency with `categoryOutOfPolicy` and also becausse `no longer valid` message on its own is not clear

Reply:
Same for tax